dashboard/app/models/teacher_feedback.rb

```diff
@@ -150,6 +154,51 @@ def summarize(is_latest = false)
     }
   end
 
+  # Passing level, script_level and student into summarize_for_csv since it's
+  # called in a loop to avoid N+1s
```

Comment:
Not sure I understand this comment.  This header comment doesn't seem to describe how this method works or what parameters it expects?

Reply:
I didn't want a future developer to remove the student and level parameters and access the value from the active record itself because while that would work, it would introduce more queries. Maybe that's an irrational fear and a developer would take into account why we're passing the parameters in instead of accessing them from the active record. I'll remove this comment and verify we have test coverage for number of queries that would catch if a developer refactored this to introduce an N+1

Reply:
Ok I've verified that the query count test I wrote previously was able to catch the additional queries that would be introduced with a refactor.